DronePan/Controllers/CameraController.swift

```diff
@@ -337,4 +341,19 @@ class CameraController: NSObject, DJICameraDelegate {
             self.status = newState
         }
     }
+    
+    func camera(camera: DJICamera, didUpdateCurrentExposureValues values: DJICameraExposureParameters) {
+        DDLogVerbose("Camera Controller didUpdateCurrentExposureValues")
+        
+        camera.getExposureModeWithCompletion(){
+            (mode, error) in
+            if error != nil {
+                DDLogWarn("ERROR: getExposureModeWithCompletion:. \(error!.description)")
```

Comment:
I've just fixed this for the code in the mixed_mission_gimbal_yaw branch - we should make sure that when logging an error object - it should be with DDLogError.

Also - to fit in with existing code - I've been using if let rather than if != nil - so this could be

```
if let error = error {
  DDLogError("getExposureModeWithCompletion - error: \(error)")
}
```

I know the effect is the same - but it would be more consistent with the existing codebase (I hope - I'm sure there's places I've missed on this too)